Submitty/Submitty
# [Refactor:API] Migration for Semester to Term

### What is the current behavior?
Currently, in the code, we use the word "term" and "semester" interchangeably. Our goal is to shift just to using "term" to be less confusing for schools without a semester system.

### What is the new behavior?
The database is now free of "semester." "semester" still appears in the code but no longer in the database.

### How to test:
Navigate to various pages on Submitty and see if anything breaks. I've successfully vagrant up'd from scratch on this branch (as well as of course migrated to it from a vagrant up on ``main``), and I've done a lot of testing so it should be good, but more testing is a great thing.
